addons/modules/stringtable.xml

```diff
@@ -1252,120 +1374,140 @@
             <Japanese>射撃パラメータ</Japanese>
+            <Korean>사격 함수</Korean>
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleFireMission_Units">
             <English>Units</English>
             <German>Einheiten</German>
             <Polish>Jednostki</Polish>
             <Japanese>ユニット数</Japanese>
+            <Korean>유닛들</Korean>
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleFireMission_">
             <English>Number of units involved in the fire mission.</English>
             <German>Anzahl der Einheiten die an der Feuermission beteiligt sind.</German>
             <Polish>Liczba jednostek wykonujących misję ogniową</Polish>
             <Japanese>砲撃ミッションをするユニット数を決定します。</Japanese>
+            <Korean>포사격 임무에 포함된 유닛의 숫자입니다.</Korean>
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleFireMission_Ammo">
             <English>Ammo</English>
             <German>Munition</German>
             <Polish>Amunicja</Polish>
             <Japanese>弾種</Japanese>
+            <Korean>탄 종류</Korean>
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleFireMission_Rounds">
             <English>Rounds</English>
             <German>Schüsse</German>
             <Polish>Pociski</Polish>
             <Japanese>弾数</Japanese>
+            <Korean>탄 갯수</Korean>
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleFireMission_Rounds_Tooltip">
             <English>Number of rounds that should be fired by each unit.</English>
             <German>Anzahl der Schüsse, die von jeder Einheit abgegeben werden sollen.</German>
             <Polish>Liczba pocisków do wystrzelenia przez każdą jednostkę.</Polish>
             <Japanese>各ユニットが発射する弾数を決定します。</Japanese>
+            <Korean>각 유닛에 의해 발사될 탄의 갯수입니다.</Korean>
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleWeather">
             <English>Change Weather</English>
             <German>Wetter ändern</German>
             <Polish>Zmień pogodę</Polish>
             <Japanese>天気変更</Japanese>
+            <Korean>날씨 변경</Korean>
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleWeather_Overcast">
             <English>Overcast</English>
             <German>Bewölkung</German>
             <Polish>Pochmurność</Polish>
             <Japanese>雲の量</Japanese>
+            <Korean>흐림</Korean>
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleWeather_Overcast_Tooltip">
             <English>Sets the amount of cloud cover, higher values result in a greater likelihood of rain/storms and faster wind speeds.</English>
             <German>Legt die Bedeckung der Wolken fest, höhere Werte führen zu einer höheren Wahrscheinlichkeit von Regen/Stürmen und schnelleren Windgeschwindigkeiten.</German>
             <Polish>Ustala pochmurność, wysoka wartość powoduje większe prawdopodobieństwo deszczu/burzy i szybszego wiatru.</Polish>
             <Japanese>雲が覆う量を決定します。高い値では大雨/嵐のようになり、強い風が吹きます。</Japanese>
+            <Korean>ADAADADAD</Korean>
```

Comment:
What's this?

Reply:
I'm sorry for my miss. I am slept at that time. I will fix it ASAP

Reply:
Fixed.